_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,220 @@
+  <<compatibility_table,version compatibility table>> you can use as a reference to know which Gruntwork Repo version
+  tag is compatible with CIS AWS 1.3.0, as well as the manuals step you need to perform to achieve said compliance.
+
+
+[[core_concepts]]
+== Core Concepts
+
+=== Background
+
+The 1.3.0 version of the CIS AWS Foundations Benchmark was released in September of 2020. You can refer to the
+https://www.cisecurity.org/benchmark/amazon_web_services/[CIS website] where you can download the latest version of the
+Benchmark (as well as all the previous versions). The latest version introduces several new recommendations. This guide
+will walk you through implementing these using Gruntwork's Infrastructure as Code Library
+so that your infrastructure is fully compliant with version 1.3.0 of the framework.
+
+==== New recommendations
+These are the new recommendations introduced in version 1.3.0 of the Benchmark. You can think of these as a "diff"
+between versions 1.2.0 and 1.3.0:
+
+- 1.19: Ensure that all the expired SSL/TLS certificates stored in AWS IAM are removed
+- 1.20: Ensure that S3 Buckets are configured with 'Block public access (bucket settings)'
+- 1.21: Ensure that the IAM Access analyzer is enabled
+- 2.1.1: Ensure all S3 buckets employ encryption-at-rest
+- 2.1.2: Ensure S3 Bucket Policy allows HTTPS requests
+- 2.2.1: Ensure EBS volume encryption is enabled
+- 3.10: Ensure that object-level logging for write events is enabled for (CloudTrail) S3 bucket
+- 3.11: Ensure that object-level logging for reading events is enabled for (CloudTrail) S3 bucket
+- 4.15: Ensure a log metric filter and alarm exists for AWS Organizations changes
+- 5.1: Ensure no network ACLs allow ingress from 0.0.0.0/0 to remote server administration ports
+
+==== New Gruntwork modules vs. existing modules
+To achieve compliance with the newly added 10 recommendations listed above, we created four new modules in the
+Gruntwork's Infrastructure as Code Library, and updated a bunch of existing modules. Namely: recommendations 1.19, 1.21,
+2.2.1 and 5.1 required creating new modules; the rest of the recommendations were achieved by updating existing modules.
+
+To ensure compliance with version 1.3.0 of the CIS AWS Foundations Benchmark, you'll need to follow all the
+instructions in the <<deployment_walkthrough>> section of this guide; precisely, follow Steps 1 and 2 to ensure that
+the existing modules get updated to their CIS 1.3.0 compliant versions and follow Step 3 to ensure that you install and
+configure the newly created modules.
+
+[[deployment_walkthrough]]
+== Deployment walkthrough
+
+Please follow the steps below to upgrade from the version 1.2.0 to 1.3.0 of the Benchmark. To see the actual relevant code
+changes, please refer to the Acme CIS examples https://github.com/gruntwork-io/cis-infrastructure-modules-acme/pull/6[here]
+and https://github.com/gruntwork-io/cis-infrastructure-live-acme/pull/8[here].
```

Comment:
I actually missed these references to Acme initially. Since they are _very_ useful for doing this upgrade, perhaps put them in a call-out, and use bullet points for the two PR links?

Reply:
By call-out, do you mean a `NOTE` block? If so, I've updated it in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/386/commits/801a74358dd3232cf824e34e20ac221860b3d364. LMK if you meant something else!